docs/4.0.0/rules/index.md

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
It's almost a copy. It uses different data source. This is so we would have different set of rules for 4.0 and 3.x